runner.sh

```diff
@@ -6,25 +6,18 @@ user=nsls2
 image="condaforge"
 timestamp=$(date +%Y%m%d%H%M%S)
 env_name="nsls2-analysis-2021-1.2"
-#docker build . -t conda --target build
-#docker build . -t temp-image --target export
-#docker image build . -t $user/${image}:latest \
- #                  -t $user/${image}:${timestamp}
-#docker run -it --rm -v $PWD:/build $user/${image}:${timestamp} bash -l /build/export.sh
-#docker create -ti --name temp-container temp-image bash
-#docker cp temp-container:/build .
+docker image build . -t $user/${image}:latest \
+                   -t $user/${image}:${timestamp}
```

Comment:
This will be a bit more readable/aligned.
```suggestion
docker image build . -t $user/${image}:latest \
                     -t $user/${image}:${timestamp}
```

Reply:
Okay. I made this change.